Tests/TuistKitTests/ImportFinder/GraphImplicitImportLintServiceTests.swift

```diff
@@ -0,0 +1,111 @@
+import Path
+import TuistCore
+import TuistLoader
+import TuistSupport
+import TuistSupportTesting
+import XcodeGraph
+import XCTest
+
+@testable import TuistKit
+
+final class GraphImplicitImportLintServiceTests: TuistUnitTestCase {
+    func test_TargetLintWithImports() async throws {
```

Comment:
I'd suggest making the test names a bit more clear in terms of the scenario – it's not clear to me how `test_TargetLintWithImports` is different from `test_TargetHandleWithImports`. I'd follow the same naming convention as we have in other tests (`test_target_lint_with_imports` or `test_targetLintWithImports`)